docs/maven-tutorial.md

```diff
@@ -0,0 +1,251 @@
+# OpenShift + Maven + Nexus Tutorial
+
+While developing your application with Java and Maven, you'll most likely be
+building many times. In order to shorten the build times of your
+pods, Maven dependencies can be cached in a local Nexus repository. This
+tutorial will step you through integrating a Nexus repository into your project.
+
+| Content: |
+|:----------------------------|
+|[Prerequisites](#prerequisites)|
+|[Setting up Nexus](#setting-up-nexus)|
+|[Connecting to Nexus](#connecting-to-nexus)|
+|[See Also](#see-also)|
+
+## Prerequisites
+
+This tutorial assumes that you'll be working with a project that is already set
+up for use with Maven. If you're interested in using Maven with your Java
+project, we highly recommend looking at
+[their guide](https://maven.apache.org/guides/getting-started/index.html).
+
+In addition, be sure to check your application's image for Maven mirror
+capabilities. Many images that use Maven have a 'MAVEN_MIRROR_URL' environment
+variable that you can use to simplify this process.
+
+## Setting up Nexus
+
+Download the official Nexus Docker image and use `new-app` to deploy it.
+```bash
+oc new-app sonatype/nexus
+```
+
+After the build has finished, run:
+```bash
+oc get svc
+```
+
+The output should resemble:
+```bash
+NAME      CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+nexus     172.30.170.249   <none>        8081/TCP   6s
+```
+
+Confirm that Nexus is running by navigating your browser to the new Cluster-IP
+at port 8081. The server may take a bit of time to start, be patient!
+
+> NOTE: Nexus comes pre-configured for the Central Repository, but you may need
```

Comment:
i'm not clear on how maven mirrors work, is it not going to still fallback to the primary nexus server if the mirror doesn't have the required content? (I assume this depends on how the repos are configured, but i would have hoped our maven_mirror_url logic configures the mirror as an additional repo, not the only replo, no?)